### PR TITLE
Initial Implementation

### PR DESCRIPTION
**Pull request by mcdonnnj**
#  #

## 🗣 Description ##

This PR is the initial implementation for the functionality of this Ansible role.

## 💭 Motivation and context ##

Until [cisagov/vdp-scanner-docker](https://github.com/cisagov/vdp-scanner-docker) is replaced with a cloud native implementation we need to generate regular scan results. The most reliable way to do this is to have an appropriate Docker Compose file that can be used to automatically launch this process.

## 🧪 Testing ##

I confirmed that this configuration works as expected on the target instance. Automated tests pass without issue.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
* [x] All new and existing tests pass.

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -1,2 +1,19 @@
...
+  ansible.builtin.copy:
+    src: docker-compose.yml
+    dest: /var/cyhy/vdp
+    mode: 0644
+
+- name: Create the mount point for output
+  ansible.builtin.file:
+    path: /var/cyhy/vdp/output
+    state: directory
+    mode: 0755
```
Comment (jsf9k) [inline tasks/main.yml]: The keys under the tasks here are not alphabetized.  We have been trying to do that of late.

**Review by jsf9k — APPROVED** [review]
Approved, but please consider my one suggestion.

File: tasks/main.yml (hunk 1 above)
Comment (mcdonnnj) [reply tasks/main.yml]: Ah, I started this a while ago and forgot to take that into account when I revisited this repo. Fixed in 1c70461. Thanks for the catch!

**Review by jsf9k — APPROVED** [review]

**Review by dav3r — APPROVED** [review]
Looks solid! 🍎

**Pull request merged**